msg/messages.go

```diff
@@ -0,0 +1,75 @@
+package msg
```

Comment:
I debated adding a full i18n pkg here and loading these from toml/yaml, but they all seemed a bit cumbersome at the moment.

Reply:
also want to add some templating functions around prettying terminal output (colors, bolds, etc) - [example](https://github.com/manifoldco/promptui/blob/d812e64c99f6ac1e81322468ce2f12a6f613892e/codes.go#L63-L88)